### DISCUSSION TITLE
Dear author, hope to add a Gantt Chart. Thank you.

### DISCUSSION THREAD

**Discussion by Honestup** [graphieros/vue-data-ui#69] [Ideas]
Dear author, hope to add a Gantt Chart. Thank you.

**Comment by graphieros** [comment]
Hi @Honestup 

Thank you for your nice suggestion. I will add this to the list of components to add to the library :)

**Reply by Honestup** [reply]
Thank you again. I appreciate the style of this project very much. If the Gantt chart can be refined to the minute, I would be very grateful

**Reply by graphieros** [reply]
I will see what I can do. If this component is to be shipped, it will have configuration options for time granularity :)
